index.js

```diff
@@ -155,14 +155,24 @@ Notifications.localNotification = function(details: Object) {
  */
 Notifications.localNotificationSchedule = function(details: Object) {
 	if ( Platform.OS === 'ios' ) {
-		this.handler.scheduleLocalNotification({
+		const iosDetails = {
 			fireDate: details.date.toISOString(),
 			alertBody: details.message,
-			userInfo: details.userInfo
-		});
+			userInfo: details.userInfo,
+			repeatInterval: details.repeatType
+		}
+		// ignore Android only repeatType
+		if (!details.repeatType || details.repeatType === 'time') {
+			delete iosDetails.repeatInterval;
```

Comment:
It's better to clean it up here rather than rely on the implementation of RTCPushNotificationManager.m to ignore it since the ignoring is not part of any specification.